deploy/cr.yaml

```diff
@@ -243,3 +243,10 @@ spec:
 #       - --target="2021-06-09 14:15:11-04"
 #       restore iindividual databases
 #       - --db-include=hippo
+
+  pmm:
+    enabled: false
+    image: percona/pmm-client:2.32.0
+    imagePullPolicy: IfNotPresent
+    secret: pmm-secret
```

Comment:
operator should not use pmm server helm secret. Please user `cluster1-pmm-secret` as we have it in PGOv1

Reply:
https://github.com/percona/percona-postgresql-operator/pull/313/commits/422611dd74c87af724c7bf38f628e6c5286f291a